config/environments/production.rb

```diff
@@ -58,7 +58,7 @@
     api_key: ENV.fetch('GOVUK_NOTIFY_API_KEY')
   }
   config.action_mailer.default_url_options = {
-    host: ENV.fetch('DOMAIN')
+    host: ENV.fetch('AUTHORISED_HOSTS').split(",")[0]
```

Comment:
I like the flexibility this gives us, but it feels a bit brittle:

- it will break if we accidentally add a space after the comma
- the emailing domain is implicitly set by the first entry in authorised hosts

We're also about to start using the domain name in logs to differentiate between environments.  So I think there's still value in `DOMAIN` existing. I wonder if there's a better way to do it than copy-pasting it into an ENV var, though — could we build it out of `customHostName` somehow?

Re whitespace, if we're only using it in the `config.hosts << domain` part of this file then we could guard against it with `String#strip`:

```
  ENV.fetch("AUTHORISED_HOSTS").split(",").map(&:strip).each do |domain|
```

Reply:
```
def authorized_hosts
    ENV.fetch("AUTHORISED_HOSTS").split(",").map(&:strip)
end

ENV.fetch("CUSTOM_HOST_NAME", authorized_hosts.first)
```

Reply:
Change implemented and queued in the pipeline.